Accepted/asc042.md

```diff
@@ -8,23 +8,22 @@ Donald Dalton <donald@transcriptic.com>
 We should allow creation of `Refs` with initial cover states.
```

Comment:
As mentioned below, cover states is somewhat ambiguous, let's make the reference clearer that its a specific cover-type that's on the ref itself

Reply:
Agreed!